src/scrabble.js

```diff
@@ -1,6 +1,63 @@
 class Scrabble {
```

Comment:
Not many pushes

Reply:
Made easy and good for us 😁 but when the code is a little too much , more pushes is needed